neon_audio/utils.py

```diff
@@ -62,9 +63,20 @@ def _plugin_to_package(plugin: str) -> str:
     }
     return known_plugins.get(plugin) or plugin
 
+def build_extra_dependency_list(config: Configuration, module: str, additional: List[str] = []) -> str:
+    extra_dependencies = config.get("extra_dependencies", [])
+    dependencies = additional + extra_dependencies.get("global", []) + extra_dependencies.get(module, [])
+    if config["tts"]["module"]:
```

Comment:
This is only necessary if `package_spec` isn't specified, so this block could be:
```yaml
if config["tts"].get("package_spec"):
    dependencies.append(config["tts"].get("package_spec")
elif config["tts"].get("module"):
    dependencies.append(config["tts"]["module"])
```